File: database/imports/courses.py
"""
AURO Courses Data Importer
Defines core IT and AI courses for BSc IT and MSc AI semesters.
"""

AURO_COURSES = [
    {"code": "IMBTTO306", "name": "Web Technologies & Frameworks", "program_code": "BScIT", "semester": 3, "credits": 4, "course_type": "theory", "hours_per_week": 4, "lab_required": False},
    {"code": "IIQATO301", "name": "Software Quality Assurance", "program_code": "BScIT", "semester": 3, "credits": 4, "course_type": "theory", "hours_per_week": 4, "lab_required": False},
    {"code": "MScAI-302", "name": "Reinforcement Learning & Multi-Agent Systems", "program_code": "MScAI", "semester": 3, "credits": 5, "course_type": "lab", "hours_per_week": 6, "lab_required": True},
    {"code": "CS-501", "name": "Deep Learning & Neural Networks", "program_code": "MScAI", "semester": 3, "credits": 4, "course_type": "theory", "hours_per_week": 4, "lab_required": False},
]

def load_courses():
    return AURO_COURSES

File: frontend/src/lib/mock/courses.ts
import { Course } from '@/types/courses';

export const MOCK_COURSES: Course[] = [
  { id: 'c1', code: 'IMBTTO306', name: 'Web Technologies & Frameworks', programCode: 'BScIT', semester: 3, credits: 4, type: 'theory', hoursPerWeek: 4, labRequired: false, createdAt: '2026-01-01' },
  { id: 'c2', code: 'IIQATO301', name: 'Software Quality Assurance', programCode: 'BScIT', semester: 3, credits: 4, type: 'theory', hoursPerWeek: 4, labRequired: false, createdAt: '2026-01-01' },
  { id: 'c3', code: 'MScAI-302', name: 'Reinforcement Learning & Multi-Agent Systems', programCode: 'MScAI', semester: 3, credits: 5, type: 'lab', hoursPerWeek: 6, labRequired: true, createdAt: '2026-01-01' },
];
